plugins/nginx-vhosts/commands

```diff
@@ -180,18 +28,52 @@ EOF
     fi
 
     tail "$NGINX_LOGS_ARGS" "$NGINX_LOGS_PATH"
-    ;;
+  ;;
+
+  nginx:disable)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    eval "$(config_export app $APP)"
+    if [[ -z "$NO_VHOST" ]]; then
+      config_set --no-restart $APP NO_NGINX=1
```

Comment:
Now that I think about it, once we're done with any changes we are making, I think it's worth looking through all of our "magic" config vars and prefixing them with `DOKKU_`, while deprecating the old usage. At least then we can grep for their usage.